3rdparty/readme.txt

```diff
@@ -51,7 +51,9 @@ jasper                JasPer is a collection of software
                       Copyright (c) 1999-2000 The University of British Columbia
                       Copyright (c) 2001-2003 Michael David Adams
 
-                      The JasPer license can be found in libjasper.
+                      See jasper official github repository
```

Comment:
Use proper names:
- JasPer
- GitHub

Reply:
Hey @alalek thanks for the response
I will do the necessary changes and change the branch to 3.4 now !